src/ingestion/azure_ingest.py

```diff
@@ -0,0 +1,85 @@
+from os import environ
+import yaml
+
+
+from azure.storage.blob import BlockBlobService
+
+from runners.helpers.auth import load_pkb_rsa
+from runners.helpers.dbconfig import PRIVATE_KEY, PRIVATE_KEY_PASSWORD
+from runners.helpers import db, log
+
+from snowflake.ingest import SimpleIngestManager
+from snowflake.ingest import StagedFile
+
+
+def get_timestamp(table):
+
+    timestamp_query = f"""
+        SELECT EVENT_TIME from {table}
+        order by EVENT_TIME desc
+        limit 1
+        """
+    ts = list(db.fetch(timestamp_query))
```

Comment:
since you're just getting the first  thing, do --

    ts = next(db.fetch(...), None)
    if ts:
        return ts['EVENT_TIME']